### ISSUE TITLE
Installation Errors npm ERR! enoent ENOENT: no such file or directory, rename 

### ISSUE THREAD

**Issue by itinance** [infinitered/ignite#46]
Installation is not possible and gives the errors listed below.

```
$ node -v
v5.10.1
$ npm -v
3.8.3

```
`npm install -g react-native-ignite`

```
npm WARN deprecated npmconf@2.1.2: this package has been reintegrated into npm and is now out of date with respect to npm

> react-native-ignite@0.1.9 preinstall /usr/local/lib/node_modules/.staging/react-native-ignite-a8026b56
> npm i -g yo

npm WARN deprecated npmconf@2.1.2: this package has been reintegrated into npm and is now out of date with respect to npm
/usr/local/bin/yo -> /usr/local/lib/node_modules/yo/lib/cli.js

> yo@1.7.0 postinstall /usr/local/lib/node_modules/yo
> yodoctor


Yeoman Doctor
Running sanity checks on your system

✔ Global configuration file is valid
✔ NODE_PATH matches the npm root
✔ Node.js version
✔ No .bowerrc file in home directory
✔ No .yo-rc.json file in home directory
✔ npm version

Everything looks all right!
/usr/local/lib
└── yo@1.7.0 

/usr/local/lib
└── (empty)

npm ERR! Darwin 15.3.0
npm ERR! argv "/usr/local/Cellar/node/5.10.1/bin/node" "/usr/local/bin/npm" "install" "-g" "react-native-ignite"
npm ERR! node v5.10.1
npm ERR! npm  v3.8.3
npm ERR! path /usr/local/lib/node_modules/.staging/abbrev-ec010306
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall rename
npm ERR! enoent ENOENT: no such file or directory, rename '/usr/local/lib/node_modules/.staging/abbrev-ec010306' -> '/usr/local/lib/node_modules/react-native-ignite/node_modules/abbrev'
npm ERR! enoent ENOENT: no such file or directory, rename '/usr/local/lib/node_modules/.staging/abbrev-ec010306' -> '/usr/local/lib/node_modules/react-native-ignite/node_modules/abbrev'
npm ERR! enoent This is most likely not a problem with npm itself
npm ERR! enoent and is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! Please include the following file with any support request:
npm ERR!     /data/workspace/npm-debug.log
npm ERR! code 1

```

The output suggest in the middle that /usr/local/lib shall be empty, but thats not true.

**Comment by GantMan** [comment]
Hey @itinance !   Thanks so much for reporting this, I'm the guilty guy who broke this 😢  The issue was, we need something installed globally, and I'm not sure WHY it's failing.

Unfortunately checking for the global install is VERY slow, so I was trying to enforce it.

Please try installing `npm i -g generator-react-native-ignite` which is the failing lib.   And please let me know if it works after this.   I'll be happy to help get this fixed ASAP.

**Comment by itinance** [comment]
Hi @GantMan! Thank you very much for fast reply :)

`npm i -g generator-react-native-ignite` works quite fine! But a following `npm install -g react-native-ignite` still fails. There is an conspicuous long waiting time on "validateTree" if this may help.
<img width="1179" alt="screen shot 2016-04-10 at 20 48 14" src="https://cloud.githubusercontent.com/assets/1758597/14412228/096f9502-ff5e-11e5-91f7-042501a24f56.png">
<img width="1251" alt="screen shot 2016-04-10 at 20 48 56" src="https://cloud.githubusercontent.com/assets/1758597/14412234/15417d0a-ff5e-11e5-9210-15d015f43ca6.png">

**Comment by GantMan** [comment]
it's my preinstall scripts.   I'm not sure why npm is so confused here, but I plan on figuring it out.   I've got to wipe some modules off my computer and then give this a fresh go again.   I'll keep you updated.

**Comment by GantMan** [comment]
@itinance - I have attempted a new fix.  It's really not easy for me to test (as when I locally symlink my libs it's not the same as it is downloading), but please try now.  I've released version 0.1.10 which should hopefully fix your install issue.

**Comment by itinance** [comment]
now it works perfectly, thx @GantMan !
